webpack/scenes/BootedContainerImages/BootedContainerImagesPage.js

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import { TableComposable, Thead, Th, Tbody, Tr, Td, ExpandableRowContent } from '@patternfly/react-table';
+import TableIndexPage from 'foremanReact/components/PF4/TableIndexPage/TableIndexPage';
+import {
+  useSetParamsAndApiAndSearch,
+  useTableIndexAPIResponse,
+} from 'foremanReact/components/PF4/TableIndexPage/Table/TableIndexHooks';
+import {
+  useUrlParams,
+  useSet,
+} from 'foremanReact/components/PF4/TableIndexPage/Table/TableHooks';
+import {
+  getColumnHelpers,
+} from 'foremanReact/components/PF4/TableIndexPage/Table/helpers';
+import {
+  useTableSort,
+} from 'foremanReact/components/PF4/Helpers/useTableSort';
+import Pagination from 'foremanReact/components/Pagination';
+import EmptyPage from 'foremanReact/routes/common/EmptyPage';
+import { translate as __ } from 'foremanReact/common/I18n';
+import BOOTED_CONTAINER_IMAGES_KEY, { BOOTED_CONTAINER_IMAGES_API_PATH } from './BootedContainerImagesConstants';
+
+const BootedContainerImagesPage = () => {
+  const columns = {
+    bootc_booted_image: {
+      title: __('Image name'),
+      isSorted: true,
+    },
+    digest: {
+      title: __('Image digests'),
+      wrapper: ({ digests }) => digests.length,
+    },
+    hosts: {
+      title: __('Hosts'),
+      wrapper: ({ bootc_booted_image: bootcBootedImage, digests }) => (
+        <a href={`/hosts?search=bootc_booted_image%20=%20${bootcBootedImage}`}>{digests.reduce((total, digest) => total + digest.host_count, 0)}</a>
```

Comment:
This needs to respect the "New host overview page" Setting. Right now it always goes to the old hosts list page.

Reply:
Ah good catch, I remember seeing this

Reply:
In fact I think I hit it during the community demo :)